source/CodeGenerator/ParameterBlock.php

```diff
@@ -58,11 +62,14 @@ public function dump() {
         if ($this->_type) {
             $content .= $this->_getType() . ' ';
         }
+        if ($this->_variadic) {
+            $content .= '...';
+        }
         if ($this->_passedByReference) {
             $content .= '&';
         }
         $content .= '$' . $this->_name;
-        if ($this->_optional) {
+        if ($this->_optional && !$this->_variadic) {
```

Comment:
Shall we rather throw ? If variadic parameter has default value? Or is optional?